Chore/initial set up.v1

NWSchallock opened the pull request:
PTAL

NWSchallock committed:
Basic Setup For Storm Breaker.

tamiavalledor1 committed:
Basic Setup

tamiavalledor1 committed:
Made Subsystems

created all subsystems and elevator has motion magic

tamiavalledor1 committed:
Made Elevator Commands

tamiavalledor1 committed:
SetUp stuff

tamiavalledor1 committed:
More Setup Stuff

duane701 reviewed (changes requested):
FTP

duane701 committed:
Ignore language.settings

duane701 committed:
Link with CTRE_Phoenix

NWSchallock committed:
Fixed name change and include error in OI.cpp

tamiavalledor1 committed:
Created SetClimber and SetIntake Commands. Updated Intake Subsystem.

duane701 reviewed (changes requested):
The last commit added some errors. 

Please fix these errors.

tamiavalledor1 committed:
Fixed errors on SetClimber command and Climber subsystem.

duane-etagen committed:
Climber: Fix typos

Missing frc namespace for DoubleSolenoid::kForward
Missing Commands directory for SetClimber.h

tamiavalledor1 committed:
Add Intake add IsCubeIn
